deepchem/feat/rdkit_grid_featurizer.py

```diff
@@ -467,38 +451,121 @@ def compute_pi_stack(protein_xyz,
             for each atom in ligand and in protein:
               add to list of atom indices
   """
-  raise ValueError("Not Yet Implemented")
-
-
-def is_cation_pi(cation_position, ring_center, ring_normal):
+  protein_pi_parallel = Counter()
+  protein_pi_t = Counter()
+  ligand_pi_parallel = Counter()
+  ligand_pi_t = Counter()
+
+  protein_aromatic_rings = []
+  ligand_aromatic_rings = []
+  for mol, ring_list in ((protein, protein_aromatic_rings),
+                         (ligand, ligand_aromatic_rings)):
+    aromatic_atoms = set(atom.GetIdx() for atom in mol.GetAromaticAtoms())
```

Comment:
You can use set notation here instead of an explicit contrutor

```python
{atom.GetIdx() for atom in mol.GetAromaticAtoms()}
```

Reply:
+1